src/js/components/modals/AppModalComponent.jsx

```diff
@@ -115,13 +106,9 @@ var AppModalComponent = React.createClass({
   },
 
   onInfoChange: function () {
-    this.removeInfoListeners();
-
-    if (!this.hasInfoObject()) {
-      this.setState({
-        info: InfoStore.info
-      });
-    }
+    this.setState({
```

Comment:
InfoStore.info always returns an (empty)-object, never null.
```JavaScript
if (!this.hasInfoObject()) {
  InfoStore.once(InfoEvents.CHANGE, this.onInfoChange);
  InfoStore.once(InfoEvents.REQUEST_ERROR, this.onInfoChange);
  InfoActions.requestInfo();
}
```
The if-condition says that the request error handler can be only triggered if the info-object is empty.
So in this case it does't make sense to re-set an an state-object on error that is already set in getInitialState. It leads to an unnecessary rerender.